### DISCUSSION TITLE
Suggestions to improve GitHub

### DISCUSSION THREAD

**Discussion by datteroandrea** [community/community#21831] [New to GitHub]
<p>It has passed a lot of time since the last GitHub website UI update and i find the new UI still not very intuitive, things i currently don’t like are:</p>
<ul>
<li>The confusion in the repo area between private and public repos i would like to make those separated</li>
<li>The main section is way to large compared to the repo section</li>
<li>The section of “expore repositories” almost useless</li>
<li>etc.</li>
</ul>
<p>So i would like to (if it’s possible):</p>
<ol>
<li>Make a GitHub page regaurding the Community Feedbacks so people can give suggestions about the general service with an upvote system so pepole can choose which is the most important suggestion to be first implemented.</li>
<li>Make the UI better and more intuitive (i can also help if you give the possibility of doing so)</li>
</ol>
<p>I know this is just an enormous thing to do, but i am only doing this to improve and help a service i LOVE.</p>
<p>I am writing here cause i don’t know where to ask elsewhere, if you know where is should ask, please tell me where i should and i will delete this post.</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! Thanks for sharing your thoughts. I suggest getting in contact with GitHub here: <a href="https://support.github.com" rel="nofollow noopener">https://support.github.com/contact</a></p>
<p>Per the main forum page: “If you want to submit a feature request or feedback about either the GitHub Community Forum or GitHub itself, please use our <a href="https://github.com/contact" rel="nofollow noopener">contact form</a>.”</p>

**Comment by tim-wharton** [comment]
The biggest thing that will fix github and make it user friendly:

UPLOAD & DOWNLOAD BUTTONS!!!! 

How are these not a thing? 

I'm sure for larger projects, using whatever fustercluck that is currently in place works well. But for the rest of us, these simple buttons are the most basic of basic of basic UI, and it is unbelievable to me that this site ever got off the ground without them.

The ONLY reason I'm using this site is because I have to for school. The number of things I've wanted to use that only exist on gitbub, and therefor not got is insane, because there is no download button!

**Reply by Ammar-Alnagar** [reply]
You can download projects as zip files or download them using github desktop and you can upload files by pressing the "upload files" button.
Ofcourse there's the git push and clone but no gui for them to my knowledge.
